iac-extensions/terraform/src/test/resources/checks/PublicAccessCheck/gcp_kubernetes_resource.tf

```diff
@@ -0,0 +1,56 @@
+# Noncompliant@+1 {{Omitting private_cluster_config grants public access to parts of this cluster. Make sure it is safe here.}}
```

Comment:
I am curious how this test PASS-es when it seems that you're not formatting the correct message in the code?